.github/workflows/nodejs.yml

```diff
@@ -181,6 +181,10 @@ jobs:
       - name: Run typings tests
         run: npm run test:typescript
 
+  test-llhttp-integrity:
+    name: Ensure integrity of llhttp wasm files
+    uses: nodejs/undici/.github/workflows/llhttp-wasm-integrity.yml@main
```

Comment:
I'm not sure I would run this for every PR. Maybe only if those files were changed?

Reply:
It is a reusable workflow. It has two steps. First check if wasm related files were changed, if so run the integrity check. Maybe the first step can be further optimized by using https://github.com/tj-actions/changed-files so that we can also take care of cases where we directly push into main. Also makes sense to obligatory run the integrity check in the automated release workflow.

Reply:
Maybe we can just run it in the release workflow then.

Reply:
@mcollina

Imho we should ensure that the repo has the correct content at any time. So if somebody tries to upstream malicious code into the repo, then it should be imho detected early and not as the last step of the release workflow. 

The only issue i see, is that people could force push into main and bypass the ci.

Reply:
we can limit that now that we have automated releases, open a separate issue.